ts/kpt-functions/src/types.ts

```diff
@@ -166,6 +166,29 @@ export class Configs {
     return this.functionConfig;
   }
 
+  /**
+   * Returns the functionConfig data field if defined.
```

Comment:
It does more than that. Look at docstring for `getFunctionConfigValue` for inspiration

Reply:
Updated doc